External/Tools/FDBuild/Program.cs

```diff
@@ -47,7 +47,7 @@ public static int Main(string[] args)
             if (File.Exists(Path.Combine(swfmillPath, "swfmill.exe")))
                 SwfmillLibraryBuilder.ExecutablePath = Path.Combine(swfmillPath, "swfmill.exe");
             else
-                SwfmillLibraryBuilder.ExecutablePath = "swfmill.exe"; // hope you have it in your environment path!
+                SwfmillLibraryBuilder.ExecutablePath = "swfmill"; // hope you have it in your environment path!
```

Comment:
Just noticed this part... shouldn't it be better that you add better checks using the swfmillPath? I'd say so... more chances of the people having swfmill there than in an environment path.